### PR TITLE
Issue #24: Throw exception if empty test plan list.

### PR DESCRIPTION
**Pull request by Jeffset**
Throw exception if empty test plan list in SingleInstrumentalTestCommand.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -18,7 +18,7 @@ plugins {
 }
 
 group 'com.github.grishberg'
-version '1.2.0'
```
Comment (Nexen23) [inline build.gradle]: major + minor versions bump at once? O_o

File: src/main/java/com/github/grishberg/tests/commands/SingleInstrumentalTestCommand.java [hunk 2]
```diff
@@ -44,6 +44,9 @@ public SingleInstrumentalTestCommand(String projectName,
     }
 
     private void initTargetTestArgs(List<TestPlanElement> testForExecution) {
+        if (testForExecution.isEmpty()) {
+            throw new IllegalArgumentException("Command is created with an empty test plan list");
```
Comment (Nexen23) [inline src/main/java/com/github/grishberg/tests/commands/SingleInstrumentalTestCommand.java]: There should be message of error, not statement of the fact. E.g. "Test plan must not be empty"

File: build.gradle (hunk 1 above)
Comment (Jeffset) [reply build.gradle]: @Grigory-Rylov asked to do it this way. He is planning to merge this after he merges 1.3.0 into master.

**Comment by Grigory-Rylov** [comment]
I don't know why CI fails, I've opened new PR with test and green build on CI

**Pull request closed without merging**